### DISCUSSION TITLE
fine control of container settings

### DISCUSSION THREAD

**Discussion by DaniDipp** [pelican-dev/panel#11] [Ideas]
I would like to see better management of the docker containers - especially being able to modify the container name, environment variables and labels.
The labels especially would be great for integration with my traefik reverse proxy.

**Comment by TekExplorer** [comment]
i'd improve the title with some of that

**Comment by parkervcp** [comment]
I updated the title to match closer to what you are asking.

Container names are used for tracking purposes and are probably not changing.

All containers variables are set via the egg. So if you want a specific variable then you can add that there.

Adding a label to a container is probably the easiest thing here to do.

**Reply by TekExplorer** [reply]
Labels I think are the biggest thing.
There's a bunch of things that could go into the labels to describe what the container is like "its a pelican server" and "has these egg features/tags" and so on

**Comment by DaniDipp** [comment]
how about adding containers to external docker networks?

**Comment by lancepioch** [comment]
Thanks, please follow #248
